GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -503,6 +503,18 @@ - (void)testRestorePreviousSignInWhenSignedOut {
   [_authState verify];
 }
 
+- (void)testNotRestorePreviousSignWhenCompletionIsNil {
+  [[[_authorization expect] andReturn:_authState] authState];
+  [[[_authState expect] andReturnValue:[NSNumber numberWithBool:NO]] isAuthorized];
+  _completionCalled = NO;
+  _authError = nil;
+
+  [_signIn restorePreviousSignInWithCompletion:nil];
+  [NSThread sleepForTimeInterval:1.0];
```

Comment:
In general, it's not good practice to sleep. Also, I don't think this is needed here. I removed this line and ran the test repeatedly (100 times) and didn't have any failure.

You can control-click on the diamond next to a test and see the following below.

![Screenshot 2023-04-20 at 11 07 19 AM](https://user-images.githubusercontent.com/4029922/233451451-10ea432b-6596-4153-a8e5-66892fde368a.png)

![image](https://user-images.githubusercontent.com/4029922/233451349-a2a877d3-c399-4d13-b409-f959adddc8b4.png)